README.md

```diff
@@ -75,7 +75,7 @@ See the `/config` folder for example configuration, used in development.
 
 ## Development
 
-This codebase has been tested with Node version 16.4.2.
+This codebase has been tested with Node version 16.16.0.
```

Comment:
I was getting weird npm errors when running tests with npm 16.14, but upgrading to 16.16 fixed that.